### PR TITLE
Additional resource cleanup from automation

### PR DESCRIPTION
**Pull request by tas50**
Add more common resource sections
Cleanup some formatting
Set more required properties

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: chef_master/source/resource_apt_update.rst [hunk 1]
```diff
@@ -228,7 +228,7 @@ Examples
 =====================================================
 .. tag resources_common_examples_intro
 
-The following examples demonstrate various approaches for using resources in recipes. If you want to see examples of how Chef uses resources in recipes, take a closer look at the cookbooks that Chef authors and maintains: https://github.com/chef-cookbooks.
+The following examples demonstrate various approaches for using resources in recipes.
```
Comment (kagarmoe) [inline chef_master/source/resource_apt_update.rst]: Should end `recipes:`

File: chef_master/source/resource_ohai.rst [hunk 2]
```diff
@@ -48,114 +47,162 @@ Properties
...
-   Continue running a recipe if a resource fails for any reason.
+  Continue running a recipe if a resource fails for any reason.
 
-``notifies``
-   **Ruby Type:** Symbol, 'Chef::Resource[String]'
+``retries``
+  **Ruby Type:** Integer | **Default Value:** ``0``
 
-   .. tag resources_common_notification_notifies
+  The number of times to catch exceptions and retry the resource.
```
Comment (kagarmoe) [inline chef_master/source/resource_ohai.rst]: Consider: "The number of attempts to catch..."

Comment (tas50) [reply chef_master/source/resource_ohai.rst]: Updated

File: chef_master/source/resource_apt_update.rst (hunk 1 above)
Comment (tas50) [reply chef_master/source/resource_apt_update.rst]: Updated

**Review by kagarmoe — APPROVED** [review]

**Pull request merged**